baremaps-core/src/main/java/org/apache/baremaps/iploc/IpLoc.java

```diff
@@ -23,9 +23,9 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 import org.apache.baremaps.geocoder.Geocoder;
-import org.apache.baremaps.geocoder.IsoCountriesUtils;
-import org.apache.baremaps.geocoder.Request;
-import org.apache.baremaps.geocoder.Response;
+import org.apache.baremaps.geocoder.request.Request;
```

Comment:
The request and response packages will never contain more than one or two classes. Should we group them?

Reply:
Response actually contains 3 classes. What would you call the package if we grouped both of them?